cmd/kubernetes-cronhpa-controller/main.go

```diff
@@ -78,3 +86,7 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func init() {
+	flag.BoolVar(&enableLeaderElection, "enableLeaderElection", false, "default not , if enable the cronHPA is primary and standby mode(currently cold standby)")
```

Comment:
Pls change `default not , if enable the cronHPA is primary and standby mode(currently cold standby)`  to 
`default false, if enabled the cronHPA would be in primary and standby mode.`

Reply:
ok